charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -46,15 +46,6 @@ data:
     Header merge Vary Origin
 
     ProxyTimeout ${PROXY_TIMEOUT}
-    OIDCOAuthTokenIntrospectionInterval 60
-    # Enabling cache encryption via 'OIDCCacheEncrypt On' has two side effects:
-    # 1. it reduces key size to avoid "could not construct cache key since key size is too large" cache errors
-    # 2. it causes intermittent duplicate requests between the proxy and the underlying app
-    # the latter side effect is too dangerous, so cache encryption is off
-    OIDCCacheEncrypt Off
-    # Use GET for the introspection endpoint to avoid a performance issue with POST requests.
-    # See https://broadworkbench.atlassian.net/browse/PROD-634
-    OIDCOAuthIntrospectionEndpointMethod GET
```

Comment:
The old mod_auth_openidc plugin had directives OIDC*, but we are now using mod_oauth2, configured [here](https://github.com/broadinstitute/datarepo-helm/blob/4db185f7d21f815f9e594f584c927ee6176ecf50/charts/oidc-proxy/templates/configmap.yaml#L214-L220)

Reply:
Just worth checking on your deployment: is it able to run with a google login (as opposed to B2C).  You can test that easily from the swagger page

Reply:
![Screenshot 2024-03-07 at 4 22 48 PM](https://github.com/broadinstitute/datarepo-helm/assets/79769153/7f419168-b2d1-4028-aa60-2170af33a6da)

Confirmed!  I can log in with googleoauth on my deployment's Swagger, and click tested with the dataset creation endpoint.